src/app/loadout-builder/ModPicker.tsx

```diff
@@ -0,0 +1,291 @@
+      ['seasonal']: 'LB.Seasonal'
+    };
+
+    const order = [...Object.values(Armor2ModPlugCategories), 'seasonal' as 'seasonal'].map(
+      (category) => ({
+        category,
+        nameTranslation: Armor2ModPlugCategoriesTitles[category]
+      })
+    );
+
+    // Only some languages effectively use the \b regex word boundary
+    const regexp = ['de', 'en', 'es', 'es-mx', 'fr', 'it', 'pl', 'pt-br'].includes(language)
+      ? new RegExp(`\\b${escapeRegExp(query)}`, 'i')
+      : new RegExp(escapeRegExp(query), 'i');
+
+    const queryFilteredMods = query.length
+      ? mods.filter(
+          (mod) =>
+            regexp.test(mod.displayProperties.name) ||
+            regexp.test(mod.displayProperties.description)
+        )
+      : mods;
+
+    const getByPlugCategoryHash = (plugCategoryHash: number) =>
+      queryFilteredMods
+        .filter((mod) => mod.plug.plugCategoryHash === plugCategoryHash)
+        .map((mod) => ({ mod, category: plugCategoryHash }));
+
+    const queryFilteredSeasonalMods = queryFilteredMods
+      .filter((mod) => specialtyModSocketHashes.includes(mod.plug.plugCategoryHash))
+      .map((mod) => ({ mod, category: 'seasonal' as 'seasonal' }));
+
+    const footer = Object.values(lockedArmor2Mods).some((f) => Boolean(f?.length))
+      ? ({ onClose }) => (
+          <ModPickerFooter
+            defs={defs}
+            categoryOrder={order}
+            lockedArmor2Mods={lockedArmor2Mods}
+            isPhonePortrait={isPhonePortrait}
+            onSubmit={(e) => this.onSubmit(e, onClose)}
+            onModSelected={this.onModSelected}
+          />
+        )
+      : undefined;
+
+    return (
+      <Sheet
+        onClose={onClose}
+        header={
+          <ModPickerHeader
+            categoryOrder={order}
+            query={query}
+            scrollToBucket={this.scrollToBucket}
+            onSearchChange={(e) => this.setState({ query: e.currentTarget.value })}
+            isPhonePortrait={this.props.isPhonePortrait}
+          />
+        }
+        footer={footer}
+        sheetClassName="item-picker"
+      >
+        <div ref={this.itemContainer} style={{ height }}>
+          {Object.values(Armor2ModPlugCategories).map((category) => (
+            <ModPickerSection
+              key={category}
+              mods={getByPlugCategoryHash(category)}
+              defs={defs}
+              locked={lockedArmor2Mods[category]}
+              title={Armor2ModPlugCategoriesTitles[category]}
+              category={category}
+              maximumSelectable={category === Armor2ModPlugCategories.general ? 5 : 2}
+              energyMustMatch={category !== Armor2ModPlugCategories.general}
+              onModSelected={this.onModSelected}
+            />
+          ))}
+          <ModPickerSection
+            mods={queryFilteredSeasonalMods}
+            defs={defs}
+            locked={lockedArmor2Mods.seasonal}
+            title="LB.Seasonal"
```

Comment:
Translate the string right here to avoid the wrath of the i18next-scanner

Reply:
Done, its the first time I had to deal with the scanner doing that. It was annoying, but I see the benefit.